generate_test.R

```diff
@@ -5,22 +5,57 @@
 
 # default option is RCP 4.5, SSP1, eta of 1 and bhm damage function
 # change to option to choose different values? But how to do so within a source function
-source(generate_cscc_test.R)
+
+# change ssp, rcp, dam_func, eta
+
+'usage: generate_test.R -s <ssp> -c <rcp> [ -r <runid> -p <type> -l <clim> -e <eta> -f <name> -t <opt>] [-a] [-o] [-d] [-w]
+options:
+ -s <ssp>   SSP baseline (random(default), SSP1, SSP2,..., SSP5)
+ -c <rcp>   RCP (random(default), rcp45, rcp60, rcp85)
+ -r <runid> Bootstart run for the damage function parameter, 0 is estimates (0<=id<=1000)
+ -p <type>  projection type (constant (default),horizon2100)
+ -l <clim>  climate models (ensemble (default), mean[-ensemble])
+ -e <eta>   elasticity of marginal utility of consumption. eta is 1 (default) or 2
+ -o         does not allow for out-of-sample damage prediction (default, allows)
+ -d         rich/poor damage function specification (default, pooled)
+ -a         5-lag damage function specification (default, 0-lag)
+ -f <name>  damage function (default=bhm (Burke et al.), djo (Dell et al.)), dice (Nordhaus)
+ -t <opt>   allows for generating tests with different temperature input. t0 (zero temp change) or t1 (1 degree for one year in one country)
+ -w         save raw data' -> doc
+
+# specify test options
+test_options <- docopt(doc, "-s SSP1 -c rcp45 -w -e 1 -t t1 -f bhm") # Default case
+
+generate_test = TRUE
+source("generate_cscc.R")
+
+if (opts["-f"] == "bhm"){
+  value = 0.08361
```

Comment:
It would be a good idea to write down the calculation you used to derive these values in case we want to update the models with other parameters

Reply:
Would I write down the calculation "by hand" or in a document and keep it somewhere or would I have to include it within the code as a comment?

Reply:
Just put it as a comment above the numbers. It may be more than one line for the growth-based terms.